src/drivers/virtio/net.zig

```diff
@@ -92,6 +92,8 @@ const VirtioNet = struct {
             self.virtio.transport.notifyQueue(self.receiveq());
         }
 
+        self.transmitq().avail.flags().* = common.VIRTQ_AVAIL_F_NO_INTERRUPT;
```

Comment:
It disables interrupts that is caused when the device consumes descriptors.

ref: https://docs.oasis-open.org/virtio/virtio/v1.2/csd01/virtio-v1.2-csd01.html#x1-712000A

> /* The driver uses this in avail->flags to advise the device: don’t 
> \* interrupt me when you consume a buffer.  It’s unreliable, so it’s 
> \* simply an optimization.  */ 
> #define VIRTQ_AVAIL_F_NO_INTERRUPT      1